.golangci.yml

```diff
@@ -223,25 +223,21 @@ linters:
     # - govet
     # - errcheck
     # - staticcheck
-    # - unused
+    - unused
     # - gosimple
-    # - structcheck
-    # - varcheck
     - ineffassign
-    # - deadcode
```

Comment:
Why is this deleted? Are not we planning to enable it later?

Reply:
No, deadcode is a deprecated linter (as mentioned in the PR description) and was replaced with unused.